### PR TITLE
Remove `:type` and `:rtype` from docstrings

### LINKED ISSUE #243: Remove `:type ...:` from all docstrings
The `:type ...:` field in the docstrings is hard to maintain, sometimes confusing and generally not needed.

### PR DESCRIPTION
**Pull request by raminqaf**
Fixes #243

### CODE REVIEW

File: docs/docs/schema/pipeline.json [hunk 1]
```diff
@@ -1062,7 +1062,7 @@
             "type": "object"
         },
         "StreamsAppConfig": {
-            "description": "StreamsBoostrap app configurations.\nThe attributes correspond to keys and values that are used as values for the streams bootstrap helm chart. :params streams: Streams Bootstrap streams section",
+            "description": "StreamsBoostrap app configurations.\nThe attributes correspond to keys and values that are used as values for the streams bootstrap helm chart. :params streams: Streams Bootstrap streams section :params autoscaling:Kubernetes Event-driven Autoscaling config",
```
Comment (disrupted) [inline docs/docs/schema/pipeline.json:1065]: the params probably shouldn't be part of the description

Comment (sujuka99) [reply docs/docs/schema/pipeline.json:1065]: :thinking: How did they get there at all??

Comment (raminqaf) [reply docs/docs/schema/pipeline.json:1065]: @sujuka99 I have no Idea 😅 at it is only for the `StreamsAppConfig`

File: kpops/components/streams_bootstrap/streams/model.py [hunk 2]
```diff
@@ -219,10 +193,8 @@ class StreamsAppConfig(KafkaAppConfig):
 
     The attributes correspond to keys and values that are used as values for the streams bootstrap helm chart.
 
-    :params streams: Streams Bootstrap streams section
-    :type streams: StreamsConfig
-    :params autoscaling:Kubernetes Event-driven Autoscaling config, defaults to None
-    :type autoscaling:StreamsAppAutoScaling, optional
+    :param streams: Streams Bootstrap streams section
+    :param autoscaling:Kubernetes Event-driven Autoscaling config, defaults to None
```
Comment (disrupted) [inline kpops/components/streams_bootstrap/streams/model.py:197]: missing space
```suggestion
    :param autoscaling: Kubernetes Event-driven Autoscaling config, defaults to None
```

**Review by disrupted — CHANGES_REQUESTED** [review]

**Pull request merged**